packages/lib-core/src/runtime/PluginLoader.ts

```diff
@@ -216,7 +216,7 @@ export class PluginLoader {
       manifest =
         typeof manifestNameOrObject === 'string'
           ? await this.loadPluginManifest(resolveURL(baseURL, manifestNameOrObject))
-          : manifestNameOrObject;
+          : this.options.postProcessManifest(manifestNameOrObject);
```

Comment:
@Hyperkid123 @florkbr 

We already call `this.options.postProcessManifest` inside `this.loadPluginManifest` and I think it's better to just call it on one place.

I'd suggest to change the `loadPluginManifest` method like so:

```diff
   const response = await this.options.fetchImpl(manifestURL, { cache: 'no-cache' });
   const responseText = await response.text();
 
-  return this.options.postProcessManifest(JSON.parse(responseText));
+  return JSON.parse(responseText);
```

and then add the following line right before the validation try/catch statement:

```ts
manifest = this.options.postProcessManifest(manifest);
```

Also please update the doc for `postProcessManifest` option:

```diff
- * Post-process the plugin manifest when fetched over the network.
+ * Post-process the plugin manifest.
  *
  * By default, no post-processing is performed on the manifest.
```

Reply:
@vojtechszocs done